test/test_datasets.py

```diff
@@ -520,6 +475,104 @@ def inject_fake_data(self, tmpdir, config):
         return split_to_num_examples[config["split"]]
 
 
+class CityScapesTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.Cityscapes
+    TARGET_TYPES = (
+        "instance",
+        "semantic",
+        "polygon",
+        "color",
+        ["instance", "semantic"],
+    )
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(
+        mode=("fine",), split=("train", "test", "val"), target_type=TARGET_TYPES
+    ) + datasets_utils.combinations_grid(
+        mode=("coarse",),
+        split=("train", "train_extra", "val"),
+        target_type=TARGET_TYPES,
+    )
```

Comment:
Nit: this is hard to read

```suggestion
    ADDITIONAL_CONFIGS = (
        *datasets_utils.combinations_grid(
            mode=("fine",), split=("train", "test", "val"), target_type=TARGET_TYPES
        ),
        *datasets_utils.combinations_grid(
            mode=("coarse",),
            split=("train", "train_extra", "val"),
            target_type=TARGET_TYPES,
        ),
    )
```